.github/workflows/release.yaml

```diff
@@ -174,11 +174,23 @@ jobs:
         id: artifact_name
         run: |
           set -euo pipefail
+          echo "version=${GITHUB_REF/refs\/tags\/v/}" >> $GITHUB_OUTPUT
           cd "${{ env.ARTIFACTS_DOWNLOAD_PATH }}"
-          file_name=$(find . -maxdepth 1 -type f -name "*darwin_amd64*" -exec basename {} \;)
-          sha="$(shasum -a 256 $file_name | awk '{printf $1}')"
-          echo "brew_files_sha=$sha" >> $GITHUB_OUTPUT
-          echo "brew_release_file=$file_name" >> $GITHUB_OUTPUT
+          darwin_intel_file_name=$(find . -maxdepth 1 -type f -name "*darwin_amd64*" -exec basename {} \;)
+          darwin_intel_sha="$(shasum -a 256 $darwin_intel_file_name | awk '{printf $1}')"
+          echo "brew_intel_files_sha=$darwin_intel_sha" >> $GITHUB_OUTPUT
+          echo "brew_intel_release_file=$darwin_intel_file_name" >> $GITHUB_OUTPUT
+          darwin_arm_file_name=$(find . -maxdepth 1 -type f -name "*darwin_arm64*" -exec basename {} \;)
+          darwin_arm_sha="$(shasum -a 256 $darwin_arm_file_name | awk '{printf $1}')"
+          echo "brew_arm_files_sha=$darwin_arm_file_name" >> $GITHUB_OUTPUT
+          echo "brew_arm_release_file=$darwin_arm_file_name" >> $GITHUB_OUTPUT
+      - name: update brew formula
+        env: 
+          # Required token to create a pull request in the tap repository 
+          HOMEBREW_GITHUB_API_TOKEN: ${{ secrets.HOMEBREW_TAP_GITHUB_TOKEN }}
+          GITHUB_USER: ${{ secrets.HOMEBREW_TAP_GITHUB_USER }}
+        run: |
+          python3 scripts/gen-brew-multi-arch.py ${{ steps.artifact_name.outputs.version }} ${{ brew_intel_release_file }} ${{ brew_intel_files_sha }} ${{ brew_arm_release_file }} ${{ brew_arm_files_sha }}
```

Comment:
Need to merge manually?

Reply:
We always needed to merge manually. If you disagree, let's talk about it.